css/main.css

```diff
@@ -86,14 +86,12 @@ details {
 	display: none;
 }
 
-@media(max-width: 500px) {
-	body {
-		padding: 0;
-		width: 15em;
+@media(max-width: 600px) {
+	#input, #result {
 		text-align: left;
```

Comment:
I think the original `text-align: center` is more beautiful even on small screens, especially given the nulled select and input margins.